scripts/generate_resource.sh

```diff
@@ -0,0 +1,238 @@
+#!/bin/bash
+
+set -euo pipefail
+
+info() {
+  printf "\r\033[00;35m$1\033[0m\n"
+}
+
+success() {
+  printf "\r\033[00;32m$1\033[0m\n"
+}
+
+fail() {
+  printf "\r\033[0;31m$1\033[0m\n"
+}
+
+__help="This script is used to quickly scaffold a resource in go-tfe. Simply provide a resource name as the first argument and it will generate standard boilerplate.
+\n- The resource name cannot be plural.
+\n- The resource name cannot contain numbers or special characters.
+\nNote: This script cannot generate appropriate camelcasing for resource names.\n
```

Comment:
I did not understand what this meant until I tried the script with `PolicySet`, expecting `policy_set` files. I think it might be helpful to provide an example here to indicate what resource the script cannot generate.